handler.go

```diff
@@ -69,8 +78,8 @@ func GetStars(w http.ResponseWriter, req *http.Request) {
 	}
 	for _, it := range items {
 		it.StargazersCount = len(it.StargazersIDs)
-		if currentUser, err := getCurrentUserID(req); err == nil {
-			it.HasStarred = hasStarred(it, currentUser)
+		if user, err := getCurrentUser(req); err == nil {
```

Comment:
Just realised we should probably do this outside the for loop, otherwise we're extracting the JWT every time

Reply:
fixed!